code/game/objects/effects/portals.dm

```diff
@@ -52,6 +52,15 @@
 /obj/effect/portal/attack_tk(mob/user)
 	return
 
+/obj/effect/portal/proc/on_entered(atom/newloc, atom/movable/AM, atom/oldloc)
```

Comment:
Change `AM` to meet the guidelines and this is ready to go.

Reply:
feexed